From 5dded31a2067a9bd5520f045b63b1b034a73dedb Mon Sep 17 00:00:00 2001
From: yilisharcs <yilisharcs@gmail.com>
Date: Sat, 11 Jul 2026 03:48:54 -0300
Subject: [PATCH 2/2] dwl tile: oldest window stays master

---
 dwl.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dwl.c b/dwl.c
index 101a45f..f6e4b1c 100644
--- a/dwl.c
+++ b/dwl.c
@@ -2718,7 +2718,7 @@ tile(Monitor *m)
 	int i, n = 0;
 	Client *c;
 
-	wl_list_for_each(c, &clients, link)
+	wl_list_for_each_reverse(c, &clients, link)
 		if (VISIBLEON(c, m) && !c->isfloating && !c->isfullscreen)
 			n++;
 	if (n == 0)
@@ -2729,7 +2729,7 @@ tile(Monitor *m)
 	else
 		mw = m->w.width;
 	i = my = ty = 0;
-	wl_list_for_each(c, &clients, link) {
+	wl_list_for_each_reverse(c, &clients, link) {
 		if (!VISIBLEON(c, m) || c->isfloating || c->isfullscreen)
 			continue;
 		if (i < m->nmaster) {
-- 
2.55.0

