diff --git c/src/arguments.rs w/src/arguments.rs
index 2ae8e566..5c139dcc 100644
--- c/src/arguments.rs
+++ w/src/arguments.rs
@@ -81,6 +81,7 @@ pub struct Arguments {
   )]
   pub(crate) color: UseColor,
   #[arg(
+    default_value = "yellow",
     env = "JUST_COMMAND_COLOR",
     help = "Echo recipe lines in <COMMAND-COLOR>",
     long,
diff --git c/src/color.rs w/src/color.rs
index 7194d729..5cfc47c1 100644
--- c/src/color.rs
+++ w/src/color.rs
@@ -64,7 +64,7 @@ impl Color {
   }
 
   pub(crate) fn doc(self) -> Self {
-    self.restyle(Style::new().fg(Blue))
+    self.restyle(Style::new().fg(Yellow))
   }
 
   pub(crate) fn doc_backtick(self) -> Self {
