From 8710a3e782719db050a65b335243913a74d15001 Mon Sep 17 00:00:00 2001
From: yilisharcs <yilisharcs@gmail.com>
Date: Wed, 27 May 2026 03:42:41 -0300
Subject: [PATCH] fix(undo): abort buffer replace if file is readonly

---
 src/nvim/undo.c | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/src/nvim/undo.c b/src/nvim/undo.c
index a80314ff7a..c6dcfbfe1a 100644
--- a/src/nvim/undo.c
+++ b/src/nvim/undo.c
@@ -3234,6 +3234,14 @@ void f_undotree(typval_T *argvars, typval_T *rettv, EvalFuncData fptr)
 /// Restores undo history on external change. Requires 'backup' to be enabled.
 static void u_handle_extern_change(const uint8_t *read_hash, linenr_T line_count)
 {
+  if (curbuf->b_p_ro
+    || (os_path_exists(curbuf->b_ffname)
+    && !os_file_is_writable(curbuf->b_ffname))) {
+    // no warning for this one so it won't make
+    // noise on opening help files and whatnot
+    return;
+  }
+
   if (!p_bk) {
     give_warning(_("File contents changed, cannot use undo info"), true, true);
     return;
-- 
2.53.0

