From ca57e761af79c0ce08fb56c7dc2709f1f418ccab Mon Sep 17 00:00:00 2001
From: yilisharcs <yilisharcs@gmail.com>
Date: Mon, 15 Jun 2026 20:40:36 -0300
Subject: [PATCH 1/5] Fix lint warnings

---
 fnlfmt.fnl | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/fnlfmt.fnl b/fnlfmt.fnl
index 9b23897..b48634a 100644
--- a/fnlfmt.fnl
+++ b/fnlfmt.fnl
@@ -147,7 +147,7 @@ number of handled arguments."
     (when (not= nil (. t 2))
       (table.insert out second))
     (if (. fn-forms callee)
-        (view-fn-args t view inspector indent2 start-indent out callee)
+        (view-fn-args t view inspector indent2 start-indent out)
         3)))
 
 (fn match? [callee]
@@ -194,7 +194,7 @@ number of handled arguments."
 (fn preserve-same-line? [t i indent out viewed depth]
   (and (<= (+ indent (length (table.concat out)) (length viewed)) 80)
        (<= depth 3) (not (fennel.comment? (. t (- i 1))))
-       (and (originally-same-lines? t 1 i) (not (viewed:find "\n")))))
+       (originally-same-lines? t 1 i) (not (viewed:find "\n"))))
 
 (fn view-body [t view inspector start-indent out callee]
   "Insert arguments to a call to a special that takes body arguments."
@@ -267,7 +267,7 @@ number of handled arguments."
       (view-pairwise-if t view inspector indent out)
       (not (originally-same-lines? t 2 3))
       (view-body t view inspector (+ start-indent 2) out callee)
-      (view-call t view inspector indent out callee)))
+      (view-call t view inspector indent out)))
 
 (fn newline-if-ends-in-comment [out indent]
   (when (: (. out (length out)) :match "^ *;[^\n]*$")
@@ -314,7 +314,7 @@ number of handled arguments."
       (.. (table.concat (icollect [_ c (ipairs cs)]
                           (tostring c))
                         (.. "\n" (string.rep " " indent)))
-          (.. "\n" (string.rep " " indent)) x)
+          "\n" (string.rep " " indent) x)
       x))
 
 (fn shorthand-pair? [k v]
-- 
2.54.0

