From 7a2ef38e9bb11e24aa7f003c6b26b627ef844dbe Mon Sep 17 00:00:00 2001
From: yilisharcs <yilisharcs@gmail.com>
Date: Fri, 19 Jun 2026 12:01:51 -0300
Subject: [PATCH 7/7] Do not rename lambda to unwritable character

---
 fnlfmt.fnl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fnlfmt.fnl b/fnlfmt.fnl
index 64affe8..dfb9f86 100644
--- a/fnlfmt.fnl
+++ b/fnlfmt.fnl
@@ -287,7 +287,7 @@ number of handled arguments."
 
 (local maybe-body {:-> true :->> true :-?> true :-?>> true :doto true :if true})
 
-(local renames {"#" "length" "~=" "not=" :lambda "λ"})
+(local renames {"#" "length" "~=" "not="})
 
 (fn body-form? [callee]
   (or (?. syntax callee :body-form?) (callee:find "%.with-")
-- 
2.54.0

