From 1512d594cc5cc7bb2211083b02471113ae11ba3d Mon Sep 17 00:00:00 2001
From: yilisharcs <yilisharcs@gmail.com>
Date: Sat, 4 Jul 2026 03:21:02 -0300
Subject: [PATCH 1/2] terminfo: advertise italics capability

---
 scripts/terminfo/kmscon.ti | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/scripts/terminfo/kmscon.ti b/scripts/terminfo/kmscon.ti
index 36c9b9d..8b583d3 100644
--- a/scripts/terminfo/kmscon.ti
+++ b/scripts/terminfo/kmscon.ti
@@ -27,9 +27,9 @@ kmscon|KMS Console,
 
 # Rendition (SGR)
   sgr=%?%p9%t\E(0%e\E(B%;\E[0%?%p6%t;1%;%?%p5%t;2%;%?%p2%t;4%;%?%p1%p3%|%t;7%;%?%p4%t;5%;%?%p7%t;8%;m,
-  sgr0=\E(B\E[m, bold=\E[1m, smul=\E[4m,
+  sgr0=\E(B\E[m, bold=\E[1m, smul=\E[4m, sitm=\E[3m,
   rev=\E[7m, smso=\E[7m, rmso=\E[27m,
-  rmul=\E[24m, smacs=\E(0, rmacs=\E(B,
+  rmul=\E[24m, ritm=\E[23m, smacs=\E(0, rmacs=\E(B,
 
 # Color Support (xterm-256color style)
   setab=\E[%?%p1%{8}%<%t4%p1%d%e%p1%{16}%<%t10%p1%{8}%-%d%e48;5;%p1%d%;m,
@@ -57,5 +57,5 @@ kmscon|KMS Console,
   kf9=\E[20~, kf10=\E[21~, kf11=\E[23~, kf12=\E[24~,
 
 # Miscellaneous
-  sc=\E7, rc=\E8, rs2=\Ec, tbc=\E[3g, hts=\EH, 
+  sc=\E7, rc=\E8, rs2=\Ec, tbc=\E[3g, hts=\EH,
   rep=%p1%c\E[%p2%{1}%-%db,
-- 
2.54.0

